addons/adyen_marketpay/security/ir.model.access.csv

```diff
@@ -0,0 +1,10 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
Do you think we should let every one having access to those models, maybe it is safer to call some method in sudo, when payment should be processed than letting everyone read the transactions, shareholders info, bank_account, ...

Tell me if i'm wrong but most of the informations are quite sensitives.

Reply:
Yes, you're right. I'll try to restrain access as much as possible.